rtp/src/rtcp/twcc.rs

```diff
@@ -1676,4 +1679,94 @@ mod test {
             );
         }
     }
+
+    #[test]
+    fn test_twcc_iter_correct_deltas() {
+        let twcc = Twcc {
+            sender_ssrc: 0.into(),
+            ssrc: 0.into(),
+            base_seq: 1,
+            status_count: 12,
+            reference_time: 1337,
+            feedback_count: 0,
+            chunks: [
+                PacketChunk::Run(PacketStatus::ReceivedSmallDelta, 2),
+                PacketChunk::VectorDouble(0b1101_0010_0101_0000, 7),
```

Comment:
Would like to include `PacketChunk::VectorSingle` here too, but I'm not sure how that should work. Does a `1` bit mean `ReceivedSmallDelta` in that case?